### DISCUSSION TITLE
Are you coming back?

### DISCUSSION THREAD

**Discussion by ghost** [devboloji/Infinix-Hackintosh-Opencore-Guide#27] [Q&A]
Hey bro, you said that you will not continue this project anymore, and i kinda believe you when i saw that this repo has been archived

But this repo is not archived anymore, are you gonna coming back? That will be awsome if you did

**Comment by devboloji** [comment]
@CreativeKittens I am thinking about it because there is no support for me and no contributions for this project. so If there are any contributions or if you people support me, I  may come in future....!!

**Reply by ghost** [reply]
Hope for the best for you🥳
